engines/misc/src/main/java/org/alfresco/transform/misc/transformers/TextToPdfContentTransformer.java

```diff
@@ -78,19 +91,29 @@ public class TextToPdfContentTransformer implements CustomTransformerFileAdaptor
     private static final byte BB = (byte) 0xBB;
     private static final byte BF = (byte) 0xBF;
 
+    @Value("${transform.core.misc.pdfBox.defaultFont}")
+    String pdfBoxDefaultFont;

```

Comment:
Can't this be `private`?

Reply:
Changed `pdfBoxDefaultFont` visibility to `private`.